wherehows-web/app/components/dataset-compliance.ts

```diff
@@ -130,16 +136,25 @@ const changeSetFieldsRequiringReview = arrayFilter<IComplianceChangeSet>(fieldCh
  */
 const initialStepIndex = -1;
```

Comment:
why is the initial index -1 instead of 0? I just thought that was kind of strange

Reply:
Array bounds, index starts at 0, so the first item is steps[0], -1, indicates that the wizard flow has not been initiated / is not active @cptran777